introduction/java_loops_II

```diff
@@ -0,0 +1,34 @@
+import java.lang.Math;
+import java.util.Scanner;
+import java.lang.Object;
+
+class Solution {
+  static private final int EXPONENT = 2;
```

Comment:
In expression a^n a is base and n is exponent, so this constant is actually a base.
Suggest to name it B_MULTIPLIER_BASE, as it's base for expression that is multiplied by b